dx_delete_vdb.py

```diff
@@ -21,12 +21,12 @@
   dx_delete_vdb.py --vdb aseTest
   dx_delete_vdb.py --vdb aseTest --engine myengine --single_thread False --force
 Options:
-  --vdb <vdb>              Name of the VDB to delete.
+  --vdb <vdb>               Name of the VDB to delete.
   --group_name <name>       Name of the group to execute against.
-  --single_thread <boolean> Run as a single thread. Use True if there are
+  --single_thread           Run as a single thread. Use True if there are
```

Comment:
--single_thread doesn't take any args, need to update the doc to reflect the new usage.